Configuration/Realm/Tests.xcconfig

```diff
@@ -1,10 +1,11 @@
-SUPPORTED_PLATFORMS = macosx iphonesimulator iphoneos;
+SUPPORTED_PLATFORMS = macosx iphonesimulator iphoneos appletvos appletvsimulator;
 SKIP_INSTALL = YES;
 
 INFOPLIST_FILE = Realm/Tests/RealmTests-Info.plist;
 PRODUCT_BUNDLE_IDENTIFIER = io.Realm.${PRODUCT_NAME:rfc1034identifier};
 
 LD_RUNPATH_SEARCH_PATHS[sdk=iphone*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
+LD_RUNPATH_SEARCH_PATHS[sdk=appletv*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
 LD_RUNPATH_SEARCH_PATHS[sdk=macosx*] = $(inherited) @executable_path/../Frameworks @loader_path/../Frameworks;
```

Comment:
That's possible. One way to figure this out would be to either write tests for watchOS if it's now possible, or manually try it.